Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/NodeFunctionGeneric.h

```diff
@@ -256,7 +256,8 @@ namespace ScriptCanvas
         {
             DataSlotConfiguration slotConfiguration;
 
-            slotConfiguration.m_name = AZStd::string::format("%s: %s", Data::Traits<ArgType>::GetName().data(), t_Traits::GetArgName(Index));
+            //slotConfiguration.m_name = AZStd::string::format("%s: %s", Data::Traits<ArgType>::GetName().data(), t_Traits::GetArgName(Index));
+            slotConfiguration.m_name = t_Traits::GetArgName(Index);
```

Comment:
We should remove the commented statement, since it is is been replaced the line that only copies over the Argument at the specified index